src/lsm/lsm_work_unit.c

```diff
@@ -636,7 +636,7 @@ __lsm_drop_file(WT_SESSION_IMPL *session, const char *uri)
      * Take the schema lock for the drop operation. Since __wt_schema_drop results in the hot backup
      * lock being taken when it updates the metadata (which would be too late to prevent our drop).
      */
-    WT_WITH_SCHEMA_LOCK(session, ret = __wt_schema_drop(session, uri, drop_cfg));
+    WT_WITH_SCHEMA_LOCK(session, ret = __wt_schema_drop(session, uri, drop_cfg, false));
```

Comment:
Should check_visibility be false or true here?

Reply:
check_visibility should be true only for dropping a table. This is not doing that so it should be false.